packages/react/react/src/hooks/index.ts

```diff
@@ -14,12 +15,14 @@ export { default as useIsomorphicLayoutEffect } from './useIsomorphicLayoutEffec
 export { default as useLiveAnnouncer } from './useLiveAnnouncer';
 export { default as useLocalStorageChangeDetector } from './useLocalStorageChangeDetector';
 export { default as useOutsideClickEffect } from './useOutsideClickEffect';
-export * from './usePreservedCallback';
+export { default as usePreservedCallback } from './usePreservedCallback';
 export { default as usePreservedReference } from './usePreservedReference';
 export { default as usePrevious } from './usePrevious';
-export * from './useRefEffect';
-export * from './useStorageState';
+export { default as useRefEffect } from './useRefEffect';
+export { default as useResizeObserver } from './useResizeObserver';
+export { default as useStorageState } from './useStorageState';
+export { default as useStyleSheetInjection } from './useStyleSheetInjection';
 export { default as useThrottle } from './useThrottle';
 export { default as useTimeout } from './useTimeout';
-export * from './useTimeoutQueue';
+export { default as useTimeoutQueue } from './useTimeoutQueue';
```

Comment:
I think default export looks inefficient because we should name it twice.
How about named export?

```typescript
// useSomething.ts
export default function useSomething() { ... }

// index.ts
export { default as useSomething } from "./useSomething";
```

Reply:
I totally agree with you. I'll change all hooks with named export!
I thought default export was a convention since most hooks are exported that way.